front/src/app/contents/forms/convention/formConvention.tsx

```diff
@@ -97,13 +97,12 @@ const conventionSection = (internshipKind: InternshipKind) => ({
   siret: {
     label:
       internshipKind === "immersion"
-        ? "SIRET de la structure d'accueil"
-        : "SIRET de l’entreprise où vous allez faire votre stage",
+        ? "SIRET de la structure d'accueil "
+        : "SIRET de l’entreprise",
     hintText:
       internshipKind === "immersion"
-        ? "la structure d'accueil, c'est l'entreprise, le commerce, l'association ... où vous allez faire votre immersion"
-        : "la structure d'accueil, où vous allez faire votre stage",
-    placeholder: "362 521 879 00034",
+        ? "la structure d'accueil, c'est l'entreprise, le commerce, l'association ... où le candidat va faire son immersion. Format attendu : 362 521 879 00034"
```

Comment:
```suggestion
        ? "la structure d'accueil, c'est l'entreprise, le commerce, l'association... où le candidat va faire son immersion. Format attendu : 362 521 879 00034"
```

Mais les points de suspension ici me font vraiment mal.